cmd/agent-js-worker/src/agent-rest-client.js

```diff
@@ -0,0 +1,444 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+
+import axios from 'axios';
+
+// All REST endpoints provided by agent controller
+const pkgs = {
```

Comment:
i assume there's a followup to add the did client?

Reply:
Yes, It not only about didclient`.  We also need `credentialclient`,`presentationclient` and `sdscomm`.